connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -272,8 +273,10 @@ public ConfigInfos validateConfigs(String connType, Map<String, String> connecto
             }
             ReflectionsUtil.registerUrlTypes();
             ConfigurationBuilder builder = new ConfigurationBuilder().setUrls(ClasspathHelper.forJavaClassPath());
-            Reflections reflections = new Reflections(builder);
 
+            builder.getUrls().remove(ReflectionsUtil.stringtoURL("/"));
```

Comment:
Yeah this was a silly idea, the issue should be handled at the time we construct the classpath as opposed to within the herder.